docs/topics/request-response.rst

```diff
@@ -834,10 +834,6 @@ TextResponse objects
 
     .. automethod:: TextResponse.follow_all
 
```

Comment:
This is quite the nitpick, but you should remove one more empty line, there are 3 now and there should be only 2 :slightly_smiling_face:

Reply:
Well it's minute but important.  Thanks for the remark I'll try and be more thorough next time !